### PR TITLE
Update community.yml

### PR DESCRIPTION
**Pull request by muthuannamalai12**
closes #ISSUE-NO
#979 
#### What does this PR do?
Updated Community.yml file 
#### Description of Task to be completed?
I have updated the issue-message, Pr-Message, and footer with new ones in Community.yml
#### How can this be manually tested?
This has been self-tested by me.
#### What is the relevant issue link?
https://github.com/EddieHubCommunity/support/issues/979

### CODE REVIEW

**Review by schmelto — APPROVED** [review]

**Review by vinzvinci — CHANGES_REQUESTED** [review]
What is better to call?

> Discord `chat` or Discord `server`?

**Comment by Vyvy-vi** [comment]
Discord server might be a better terminology 🤔

**Comment by DevCassie** [comment]
> Discord server might be a better terminology 🤔

Agreed.

File: .github/workflows/community.yml [hunk 1]
```diff
@@ -20,10 +20,9 @@ jobs:
...
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          issue-message: '<h1>It''s great having you contribute to this project</h1> Thank you for raising an <strong>Issue</strong> and welcome to the community :nerd_face:'
-          pr-message: '<h1>It''s great having you contribute to this project</h1> Thank you for creating a <strong>Pull Request</strong> and welcome to the community :nerd_face:'
-          footer: 'Join our <a href="http://discord.eddiehub.org">Discord</a> and <a href="http://github.eddiehub.org">GitHub</a> Organisation'
-
+          issue-message: '<h1>It''s great having you contribute to this project</h1> Welcome to the community :nerd_face:'
+          pr-message: '<h1>It''s great having you contribute to this project</h1> Welcome to the community :nerd_face:'
+          footer: 'If you would like to continue contributing to open source and would like to do it with an awesome inclusive community, you should join our <a href="http://discord.github.org">Discord</a> server and our <a href="http://github.eddiehub.org">GitHub Organisation</a> - we help and encourage each other to contribute to open source little and often 🤓 . Any questions let us know.'
```
Comment (eddiejaoude) [inline .github/workflows/community.yml:25]: Url is incorrect, here is the fix

```suggestion
          footer: 'If you would like to continue contributing to open source and would like to do it with an awesome inclusive community, you should join our <a href="http://discord.eddiehub.org">Discord</a> server and our <a href="http://github.eddiehub.org">GitHub Organisation</a> - we help and encourage each other to contribute to open source little and often 🤓 . Any questions let us know.'
```

File: .github/workflows/community.yml [hunk 2]
```diff
@@ -20,10 +20,9 @@ jobs:
...
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          issue-message: '<h1>It''s great having you contribute to this project</h1> Thank you for raising an <strong>Issue</strong> and welcome to the community :nerd_face:'
-          pr-message: '<h1>It''s great having you contribute to this project</h1> Thank you for creating a <strong>Pull Request</strong> and welcome to the community :nerd_face:'
-          footer: 'Join our <a href="http://discord.eddiehub.org">Discord</a> and <a href="http://github.eddiehub.org">GitHub</a> Organisation'
-
+          issue-message: '<h1>It''s great having you contribute to this project</h1> Welcome to the community :nerd_face:'
+          pr-message: '<h1>It''s great having you contribute to this project</h1> Welcome to the community :nerd_face:'
+          footer: 'If you would like to continue contributing to open source and would like to do it with an awesome inclusive community, you should join our <a href="http://discord.eddiehub.org">Discord</a> server and our <a href="http://github.eddiehub.org">GitHub Organisation</a> - we help and encourage each other to contribute to open source little and often 🤓 . Any questions let us know.''
```
Comment (Vyvy-vi) [inline .github/workflows/community.yml:25]:
```suggestion
          footer: 'If you would like to continue contributing to open source and would like to do it with an awesome inclusive community, you should join our <a href="http://discord.eddiehub.org">Discord server</a> and our <a href="http://github.eddiehub.org">GitHub Organisation</a> - we help and encourage each other to contribute to open source little and often 🤓 . Any questions let us know.''
```

**Review by Vyvy-vi — APPROVED** [review]
lgtm, but maybe we can hyperlink `Discord server` for more consistency?

**Review by SashoStoichkov — DISMISSED** [review]
LGTM 👍🏻 If you make the changes @Vyvy-vi requested - it'll be ready for merge 😉

**Review by SashoStoichkov — APPROVED** [review]
LGTM 👍🏻 Great work everyone

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1:

**Comment by SashoStoichkov** [comment]
Thanks @AllanRegush 👍🏻

**Pull request merged**